tiledb/cloud/rest_api/README.md

```diff
@@ -136,12 +136,15 @@ Class | Method | HTTP request | Description
  - [Array](rest_api/docs/Array.md)
  - [ArrayActions](rest_api/docs/ArrayActions.md)
  - [ArrayActivityLog](rest_api/docs/ArrayActivityLog.md)
+ - [ArrayInfo](rest_api/docs/ArrayInfo.md)
+ - [ArrayInfoUpdate](rest_api/docs/ArrayInfoUpdate.md)
  - [ArrayMetadata](rest_api/docs/ArrayMetadata.md)
- - [ArrayMetadataUpdate](rest_api/docs/ArrayMetadataUpdate.md)
+ - [ArrayMetadataEntry](rest_api/docs/ArrayMetadataEntry.md)
```

Comment:
should this be `ArrayInfoEntry`?

Reply:
No we switched the names of classes in the openapi spec. Cloud "metadata" is now called "info" to avoid confusion with core tiledb array metadata. This new `ArrayMetadataEntry` is from the serialization of the core array metadata.